README.md

```diff
@@ -15,13 +15,12 @@ where the static site is built, and then the whole thing is deployed to
 
 ### Dependencies ###
 
-You'll need to have `cabal` and `ghc` installed. If you're on OS X, you can do
+You'll need to have `stack` and `ghc` installed. If you're on OS X, you can do
 that with homebrew:
 
 ```
 % brew update
-% brew install ghc cabal-install
-% cabal install cabal-install
+% brew install ghc stack
```

Comment:
* The package is named `haskell-stack`, not `stack`
* `haskell-stack` depends on `ghc`, so you probably don't need to explicitly mention `ghc`

This line could thus be: `% brew install haskell-stack`